### PR TITLE
Added Bootstrap ReadOnlyField

### PR DESCRIPTION
**Pull request by juanmhidalgo**
Added ReadOnlyField as a shorthand to Field('username', readonly='readonly').
This is related to #341 and #326

### CODE REVIEW

**Comment by kmmbvnr** [comment]
Generally, it's not enough, b/c it's modifies only form rendering, but still allow to do direct POST with a new value.

Django 1.9 have Field.disbled option for that::
https://docs.djangoproject.com/en/1.9/ref/forms/fields/#disabled

**Comment by carltongibson** [comment]
OK. I'm happy to have this. (Is it worth the extra class? If people think so then...)

@juanmhidalgo Please add the `disabled` option as per @kmmbvnr's comment — thanks for the input! — do we need to fork on version there...?

**Comment by juanmhidalgo** [comment]
Thanks @kmmbvnr  for your comment.
I realize that adding the disabled=True to field definition is enough, at least in django 1.9.

@carltongibson  if I add the disabled attribute on older versions of django I will lose the value after submiting my form. For django 1.9 I still need to add disabled to field definition that solves my original problem.

**Comment by carltongibson** [comment]
So what's the cross version solution?

File: crispy_forms/bootstrap.py [hunk 1]
```diff
@@ -386,3 +386,9 @@ def __init__(self, field, *args, **kwargs):
 
 class InlineField(Field):
     template = "%s/layout/inline_field.html"
+
+
+class ReadOnlyField(Field):
+    def __init__(self, *args, **kwargs):
+        self.attrs = {'readonly': 'readonly'}
```
Comment (carltongibson) [inline crispy_forms/bootstrap.py]: A `if django.VERSION...` here?

Comment (juanmhidalgo) [inline crispy_forms/bootstrap.py]: It sounds good to me if you are agree.

I will modify and send again.

Thanks.

File: crispy_forms/bootstrap.py [hunk 2]
```diff
@@ -386,3 +388,9 @@ def __init__(self, field, *args, **kwargs):
 
 class InlineField(Field):
     template = "%s/layout/inline_field.html"
+
+if django.VERSION < (1, 9):
```
Comment (carltongibson) [inline crispy_forms/bootstrap.py]: Hmmm. This wasn't quite what I thought you wanted... 

If we do it this was `ReadOnlyField` won't be available at all in 1.9 — the `disabled` attribute will do all. 

Is that correct? Is it enough? I'm happy to do it this way if so. 

(I had thought in `__init__` you'd set `readonly` for 1.8 and `disabled` for 1.9+)

**Comment by carltongibson** [comment]
I'm going to close this as-is. It looks like `disabled` is sufficient for recent versions of Django. I'm happy for users to add their own `ReadOnlyField` prior to that. 

If there are specific requests beyond this we can address them in separate PRs

**Pull request closed without merging**